src/styles/accountConnection.styl

```diff
@@ -20,6 +21,21 @@
         flex-direction column
         justify-content center
 
+        .col-konnector-update
+            width 90%
+            align-self center
+            border 1px solid yourPink
+            background-color chablis
+            color monza
+            border-radius 5px
+            padding .5rem
+            cursor pointer
+
+            &:hover,
+            &:active,
+            &:focus
+                text-decoration underline
```

Comment:
Why is this necessary on this particular class ? Shouldn't all links be underlined on active, hover, focus ? Shouldn't they be underlined by default ? This is a  concerning UX issue if this is not already the case.

Reply:
It was necessary since the `<a>` didn't have a `href` attribute, but now it uses the `ButtonAction` from cozy-ui